code/game/objects/items/weapons/sharpener.dm

```diff
@@ -1,8 +1,8 @@
 /obj/item/weapon/sharpener
-	name = "whetstone"
+	name = "sharpening rod"
```

Comment:
Is this actually needed?

Reply:
it would make more sense to find a sharpening rod at the end of a super secure military base than to find a chef's whetstone.

Reply:
also i had a sprite for it